backend/api/serializers.py

```diff
@@ -52,6 +52,10 @@ def to_representation(self, instance):
         """Переопределяем метод для исключения _cls из statuses"""
         representation = super().to_representation(instance)
 
+        if "photos" in representation:
+            updated_photos = [f"http://localhost:8000/api/photos/{str(photo.id)}/" for photo in instance.photos]
```

Comment:
Хардкодить так хоста - не гибко. Опустим ситуации деплоя на домен. Если банально поменять порт, то всё накроется. Лучше сделать как-то так:
```suggestion
            updated_photos = [f"{os.getenv('URL_BASE')}/api/photos/{str(photo.id)}/" for photo in instance.photos]
```